langchain/src/document_loaders/tests/notionapi.int.test.ts

```diff
@@ -32,7 +32,7 @@ test("Test Notion API Loader Database", async () => {
   });
 
   const docs = await loader.load();
-  const titles = docs.map((doc) => doc.metadata.properties.title);
+  const titles = docs.map((doc) => doc.metadata.properties._title);
```

Comment:
Is this breaking?

Reply:
For an example:
- Non-database Pages have the `...properties.title` property still available, as well as `...properties._title`. 
- A Database Page with the title properties called `Name` would have `...properties.Name` as well as `...properties._title`.

This allows a user to always access the `_title` property, irrespective of the database schema.